streamlit_app/harness_runner.py

```
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+from evaluation.run_evaluation_harness import EvaluationHarness
+from lib.timestamp_utils import get_current_timestamp
+from constants import PROJECT_ROOT, OUTPUTS_DIR
+
+
+def _run_harness(dataset_path: Path, model: str, progress_state: dict, result_state: dict, state_lock):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_run_harness`

Add return type annotation: `None`

(ANN202)